src/NexusMods.App.UI/ViewModels/Helpers/ViewModelSelector/ViewModelDesignSelector.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reactive.Linq;
+using System.Windows.Input;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+namespace NexusMods.App.UI.ViewModels.Helpers.ViewModelSelector;
+
+/// <summary>
+/// ViewModelSelector for design time, this switches between hardcoded view models.
+/// </summary>
+/// <typeparam name="TEnum"></typeparam>
+/// <typeparam name="TVmType"></typeparam>
+/// <typeparam name="TBase"></typeparam>
+public class ViewModelDesignSelector<TEnum, TVmType, TBase> : 
+    AViewModelSelector<TEnum, TVmType, TBase>
+    where TVmType : class, IViewModelInterface
+    where TEnum : struct, Enum {
+    private static readonly Dictionary<TEnum,Type> Mappings;
+
+    static ViewModelDesignSelector()
+    {
+        Mappings = AViewModelAttribute.GetAttributes<TEnum>();
+    }
```

Comment:
The base class `AViewModelSelector` already has a protected field called `Mappings`. These lines (L18-L23) can be removed.